packages/serialize/types/index.d.ts

```diff
@@ -63,7 +61,20 @@ export interface ObjectInterpolation<MP>
   extends CSSPropertiesWithMultiValues,
     CSSPseudos<MP>,
     CSSOthersObject<MP> {}
-export type FunctionInterpolation<MP> = (mergedProps: MP) => Interpolation<MP>
+
+export interface FunctionInterpolation<MP> {
```

Comment:
MP looks enigmatic, could we rename it to MergedProps or to something more meaningful (if you have a better name than MergedProps for this)?

Reply:
Done, don't have a better name.